examples/loop_select.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+import socket
+import uuid
+import paho.mqtt.client as mqtt
+from select import select
+from time import time
+
+client_id = 'paho-mqtt-python/issue72/' + str(uuid.uuid4())
+topic = client_id
+print("Using client_id / topic: " + client_id)
+
+
+class SelectMqttExample:
+    def __init__(self):
+        pass
+
+    def on_connect(self, client, userdata, flags, rc):
+        print("Subscribing")
+        client.subscribe(topic)
+
+    def on_message(self, client, userdata, msg):
+        if self.state not in {1,3,5}:
```

Comment:
I would prefer to see the positive case first, so that the else clause doesn't mean "not not in {...}".

Reply:
I removed the "else". I prefer to have a program flow where the "good" case is always on the top indention level and all the bad cases just exit the function as soon as possible. "if this is bad, abort. If that is bad, abort. What's remaining must be good."